keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -44,12 +44,19 @@ class ApplicationController(
   )
   fun get(
     @PathVariable("application") application: String,
-    @RequestParam("entities", required = false, defaultValue = "") entities: List<String>
+    @RequestParam("includeDetails", required = false, defaultValue = "false") includeDetails: Boolean,
+    @RequestParam("entities", required = false, defaultValue = "") entities: MutableList<String>
   ): Map<String, Any> {
     return mutableMapOf<String, Any>(
       "applicationPaused" to actuationPauser.applicationIsPaused(application),
-      "hasManagedResources" to applicationService.hasManagedResources(application)
+      "hasManagedResources" to applicationService.hasManagedResources(application),
+      "currentEnvironmentConstraints" to applicationService.getConstraintStatesFor(application)
```

Comment:
Feel free to move `currentEnvironmentConstraints` into the `also` block so that it's only on the "legacy" `includeDetails` version of this route.

Reply:
It's OK -- @erikmunson and I hadn't concluded on where the right place to include constraint information in the new format would be, so let's leave it here for now.